mod/primitives/pkg/encoding/hex/bytes.go

```
@@ -39,7 +40,13 @@
 
 // MustToBytes returns the bytes represented by the given hex string.
 // It panics if the input is not a valid hex string.
-func MustToBytes(input string) []byte {
+func MustToBytes(input string) (bz []byte) {
+	defer func() {
+		if r := recover(); r != nil {
+			fmt.Errorf("failed to decode hex string: %v", r)
```

golangci-lint (1.62.2)

Error return value of `fmt.Errorf` is not checked

(errcheck)